source/material_model/visco_plastic.cc

```diff
@@ -620,7 +620,6 @@ namespace aspect
           strain_rheology.parse_parameters(prm);
 
           // Reference and minimum/maximum values
-          reference_T = prm.get_double("Reference temperature");
```

Comment:
Is there still a declaration of this variable in visco_plastic.h? If so, please remove as well.

Reply:
Good point. Removed.